MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -1,10 +1,168 @@
+    private var correctAnswers: Int = 0
+
+    @IBOutlet private weak var counterLabel: UILabel!
+    @IBOutlet private weak var imageView: UIImageView!
+    @IBOutlet private weak var questionLabel: UILabel!
+    @IBOutlet private weak var noButton: UIButton!
+    @IBOutlet private weak var yesButton: UIButton!
+
     override func viewDidLoad() {
         super.viewDidLoad()
+        show(quiz: convert(model: questions[currentQuestionIndex]))
+    }
+    
+    private func show(quiz step: QuizStepVeiwModel) {
+        counterLabel.text = step.questionNumber
+        imageView.image = step.image
+        questionLabel.text = step.question
+    }
+
+    private func convert(model: QuizQuestion) -> QuizStepVeiwModel {
+        return QuizStepVeiwModel(
+            image: UIImage(named: model.image) ?? UIImage(),
+            question: model.text,
+            questionNumber: "\(currentQuestionIndex + 1)/\(questions.count)")
+    }
+    
+    private func show(quiz result: QuizResultsVeiwModel) {
+        let allert = UIAlertController(
+            title: result.title,
+            message: result.text,
+            preferredStyle: .alert)
+        let action = UIAlertAction(title: result.buttonText, style: .default) { _ in
+            self.currentQuestionIndex = 0
+            self.correctAnswers = 0
+            let firstQuestion = questions[self.currentQuestionIndex]
+            let viewModel = self.convert(model: firstQuestion)
+            self.show(quiz: viewModel)
+        }
+        allert.addAction(action)
+        self.present(allert, animated: true, completion: nil)
+    }
+    
+    private func showNextQuestionOrResults() {
+        if currentQuestionIndex == questions.count - 1 {
+            let text = "Ваш результат: \(correctAnswers) из 10"
+            let viewModel = QuizResultsVeiwModel(
+                title: "Этот раунд окончен!",
+                text: text,
+                buttonText: "Сыграть ещё раз")
+            imageView.layer.borderColor = UIColor.clear.cgColor
+            show(quiz: viewModel)
+        } else {
+            currentQuestionIndex += 1
+            let nextQuestion = questions[currentQuestionIndex]
+            let viewModel = convert(model: nextQuestion)
+            imageView.layer.borderColor = UIColor.clear.cgColor
+            show(quiz: viewModel)
+        }
+    }
+    
+    private func showAnswerResult(isCorrect: Bool) {
+        if isCorrect {
+            correctAnswers += 1
+        }
+        self.noButton.isEnabled = false
+        self.yesButton.isEnabled = false
+        DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) {
+            self.showNextQuestionOrResults()
+            self.noButton.isEnabled = true
+            self.yesButton.isEnabled = true
+        }
+        imageView.layer.borderWidth = 8
+        imageView.layer.borderColor = isCorrect ? UIColor(named: "YP Green (iOS)")!.cgColor : UIColor(named: "YP Red (iOS)")!.cgColor
```

Comment:
Я не рекомендую использовать focreunwrapping в учебном коде (даже если это работает), это не самая лучшая черта работ будущего джуна
```suggestion
        imageView.layer.borderColor = isCorrect ? UIColor(named: "YP Green (iOS)")?.cgColor : UIColor(named: "YP Red (iOS)")?.cgColor
```

однако как константу (в расширение Colors в helper), можешь вынести статическим свойством с force unwrap'ом – для констант это в целом приемлемо